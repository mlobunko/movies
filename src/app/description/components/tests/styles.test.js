import React from 'react';
import { shallow } from 'enzyme';
import 'styled-components';

import {
  StyledDescription,
  StyledLabel,
  StyledList,
  StyledRow,
  StyledRowLeft,
  StyledRowRight,
  StyledTagline
} from '../styles';

describe('StyledDescription', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledDescription />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledLabel', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledLabel />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledList', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledList />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledRow', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledRow />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledRowLeft', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledRowLeft />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledRowRight', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledRowRight />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledTagline', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledTagline />);
    expect(wrapper).toMatchSnapshot();
  });
});
