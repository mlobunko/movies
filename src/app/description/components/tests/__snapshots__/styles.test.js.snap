// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledDescription renders correctly 1`] = `
<div
  className="sc-bdVaJa jrnoBA"
/>
`;

exports[`StyledLabel renders correctly 1`] = `
<span
  className="sc-bwzfXH dsjofN"
/>
`;

exports[`StyledList renders correctly 1`] = `
<p
  className="sc-htpNat bhpHJD"
/>
`;

exports[`StyledRow renders correctly 1`] = `
<div
  className="sc-bxivhb dYSKPg"
/>
`;

exports[`StyledRowLeft renders correctly 1`] = `
<div
  className="sc-ifAKCX kLPerT"
/>
`;

exports[`StyledRowRight renders correctly 1`] = `
<div
  className="sc-EHOje dFrzXf"
/>
`;

exports[`StyledTagline renders correctly 1`] = `
<p
  className="sc-bZQynM gXmudf"
/>
`;
